lib/spack/llnl/util/filesystem.py

```diff
@@ -1113,11 +1113,11 @@ def find(root, files, recursive=True):
     Parameters:
         root (str): The root directory to start searching from
         files (str or Sequence): Library name(s) to search for
-        recurse (bool, optional): if False search only root folder,
+        recursive (bool): if False search only root folder,
             if True descends top-down from the root. Defaults to True.
 
     Returns:
-        list of strings: The files that have been found
```

Comment:
We could probably do something like `typing.List[str]` to preserve this information, not sure if there's a simpler way. We could also just wait until we drop Python 2 support and go crazy with mypy, since this will all need to be moved anyway.